tests/unit/src/test/scala/tests/decorations/SyntheticHoverLspSuite.scala

```diff
@@ -56,7 +56,7 @@ class SyntheticHoverLspSuite extends BaseLspSuite("implicits") {
         "  \"foo\".map(c @@=> c.toUpper)",
         """|**With synthetics added**:
            |```scala
-           |scala.Predef.augmentString("foo").map[scala.Char, scala.Predef.String](c => scala.LowPriorityImplicits.charWrapper(c).toUpper)(scala.Predef.StringCanBuildFrom)
```

Comment:
Same question, there should still be a type parameter here, no?

Reply:
Alright fixed this as well. Same as the other test the `map` that was used before in 2.13 actually doesn't have a type parameter there. However when using `toDouble` I can get it back. So I've switched this to the same thing.